tests/test_nn.py

```diff
@@ -150,15 +150,27 @@ def _check_call(
     if should_be_called:
         call_count += 1
 
-        self.assertEqual(wrapped.call_count, call_count)
+        self.assertEqual(call_count, wrapped.call_count)
 
-        # called with one positional argument ...
-        self.assertEqual(1, len(wrapped.call_args.args))
+        # Lets check the tuple
+        self.assertIsInstance(wrapped.call_args, tuple)
+
+        call_size = len(wrapped.call_args)
+        # Make sure tuple at least has positional arguments, could be 3 if kwargs available
+        self.assertLessEqual(2, call_size)
 
+        if call_size == 2:
```

Comment:
@cthoyt add some comment why we need this? (and that we might get rid of it, once Python 3.7 support is dropped sometime in the future)